pepper/libpepper.py

```diff
@@ -7,9 +7,13 @@
 import functools
 import json
 import logging
-import ssl
 import os
 try:
+    import ssl
+    ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
Do you know what Python versions this assignment will work for? I'm under the impression it works for 2.6 but I'm not sure about 2.5.